web/packages/teleterm/src/sharedProcess/ptyHost/ptyProcess.ts

```diff
@@ -183,14 +199,14 @@ export class PtyProcess extends EventEmitter implements IPtyProcess {
   }
 }
 
-export const TermEventEnum = {
-  CLOSE: 'terminal.close',
-  RESET: 'terminal.reset',
-  DATA: 'terminal.data',
-  OPEN: 'terminal.open',
-  EXIT: 'terminal.exit',
-  START_ERROR: 'terminal.start_error',
-};
+export enum TermEventEnum {
```

Comment:
Refactored this in a proper enum so that it's easier to use it as the type in `addListenerAndReturnRemovalFunction`.